src/e2e-browser/roles.ts

```diff
@@ -38,15 +39,17 @@ export const essUser = Role("http://localhost:1234", async (t) => {
  * Move this function into `essUser` and replace uses of it with uses of the Role
  * once this issue is resolved: https://github.com/inrupt/solid-client-authn-js/issues/423
  */
-export const essUserLogin = async (_t: TestController) => {
+export const essUserLogin = async (
```

Comment:
```suggestion
export const userLogin = async (
```

Potentially, the tests might become more than just ESS, so maybe rename that variable?

Reply:
I'd approach this the other way around, and rename it if it became more. Right now, this is quite specific to ESS's Cognito page.

Reply:
OK, makes sense.